packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -174,24 +169,19 @@ class OptimisticOracleContractMonitor {
         `.\n Collateral currency address is ${createEtherscanLinkMarkdown(
           this.oracleType === OptimisticOracleType.OptimisticOracle ? event.currency : event.request.currency
         )}. ` +
-        `tx ${createEtherscanLinkMarkdown(event.transactionHash, this.contractProps.networkId)}`;
-
-      const logLevel =
-        this.logOverrides.proposedPrice || (this._isFundingRateIdentifier(event.identifier) ? "info" : "error");
+        `tx ${createEtherscanLinkMarkdown(event.transactionHash, this.contractProps.networkId)}. ${this._generateUILink(
+          event.transactionHash,
+          this.contractProps.networkId
+        )}.`;
 
       // The default log level should be reduced to "info" for funding rate identifiers:
-      this.logger[logLevel]({
+      this.logger[
+        this.logOverrides.proposedPrice || (this._isFundingRateIdentifier(event.identifier) ? "info" : "error")
+      ]({
         at: "OptimisticOracleContractMonitor",
         message: `${this.oracleType}: Price Proposal Alert 🧞‍♂️!`,
         mrkdwn,
-        notificationPath: "optimistic-oracle",
-      });
-
-      // UI link is sent separately because it can break slack message limits.
-      this.logger[logLevel === "error" ? "info" : logLevel]({
-        at: "OptimisticOracleContractMonitor",
-        message: `${this.oracleType}: Price Proposal Alert 🧞‍♂️!`,
-        mrkdwn: this._generateUILink(event.requester, event.identifier, event.timestamp, event.ancillaryData),
+        discordPaths: ["oo-fact-checking", "oo-events"],
```

Comment:
here you can see part of the new settings in action: we can define discord specific settings.

Reply:
These have to correspond to discord slack channels right? I would probably recommend storing these hardcoded values in a constants file or somewhere else in case channel names change. If channel names change then we only need to change the constants file and have less diff

Reply:
+1 constants would help here

Reply:
so the logic with this is that it is _only_ for discord. it does not have to be associated with slack channels (and in fact is unrelated to slack logic!). there are only a finite set of events we care about on Discord (the total in this PR).